### ISSUE TITLE
Crashing badly 

### ISSUE THREAD

**Issue by borygo77** [Tikipeter/tikipeter.github.io#165]
!

**Comment by hcgiub001** [comment]
Tikipeter has this tag
![image](https://github.com/user-attachments/assets/67ecf213-b5f9-4ab8-89ba-0d22c12ae8c1)

no please , no thank you
manners don't cost much and when you are asking others for help and to use up their personal time, I' advise you use them

**Comment by borygo77** [comment]
If you don't have anything smarter to say then shut up and quit my issue!

**Comment by hcgiub001** [comment]
you don't think it's smart to use manners when you are asking for help for no compensation? So you're saying you don't respect another person's personal time and think they should do as you say and when you been told how rude it is in order to be helpful and not disrespect someone you try to cause an issue instead of correcting your behaviour?

**Comment by Tikipeter** [comment]
No manners, sure.

But this needs to be addressed, so there is a setting to limit concurrent threads in the next update that should control this issue. If not set, concurrent threads are limited to 60.

**Comment by borygo77** [comment]
Thank you! 😊

**Comment by borygo77** [comment]
Thank you for the update Peter!
Appreciate your help but problem was somewhere else.
I made a widget from "because you watched" and if TMDB is set as provider then fenlight is throwing these errors bacause of late TMDB login. 
It takes about 10sec for TMDB to login after reboot or kodi restart... 

This doesn't happen on my Linux machine or android phone. Only coreelec got this problem..
Changed to imdb and no more errors even with limit concurrent thread disabled 😉
Happy days!

![Screenshot_20241106_111923.jpg](https://github.com/user-attachments/assets/921d77bb-8ccf-4cac-b326-a094013b6073)
